README.md

```diff
@@ -38,6 +38,8 @@ A curated list of awesome online implementations of integrated development envir
 * [Theia](https://theia-ide.org/) - [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/theia-ide/theia) [<img title="Self Hosted" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/docker.svg" />](https://hub.docker.com/u/theiaide/) - Run any language on a full VM complete with a terminal built on [Monaco Editor](https://microsoft.github.io/monaco-editor/) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/Microsoft/monaco-editor)
 * [Datalore](https://datalore.io/) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run Python notebooks for data analysis and visualization, with real-time collaboration, incremental calculation, and integrated version control
 * [MATLAB Online](https://www.mathworks.com/products/matlab-online.html) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run MATLAB code, visualize results, share scripts and collaborate on mathematical computing software
+* [Colaboratory](https://research.google.com/colaboratory/) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/googlecolab/colabtools) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run Python in a Jupyter notebook environment supporting Machine Learning libraries for education and research and free GPUs for running experiments
```

Comment:
Change "Run Python in a Jupyter notebook environment supporting Machine Learning libraries" to "Run Python in a Jupyter notebook with ML libraries"

Also, is "education and research" necessary here? I'm not familiar with Jupyter but I assume that is the typical use case, even outside of colaboratory.

Reply:
Removed the redundant information. Thanks!